static/app/views/explore/hooks/useTimeseries.tsx

```diff
@@ -0,0 +1,137 @@
+import type {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+
+/**
+ * Row represents a point in a timeseries - plotting (timestamp, value) with a line between them would represent a line chart.
+ */
+export interface Row {
+  /** timestamp: The time (in seconds) of this point in the time series */
```

Comment:
Attribute docs don't need to have the name of the attribute in them 👍🏻 The editor UI takes care of that